agent/engine/docker_task_engine_test.go

```diff
@@ -181,11 +184,14 @@ func TestBatchContainerHappyPath(t *testing.T) {
 		},
 	}
 
-	if cont := <-contEvents; cont.Status != api.ContainerStopped {
+	event = <-stateChangeEvents
+	if cont := event.(api.ContainerStateChange); cont.Status != api.ContainerStopped {
 		t.Fatal("Expected container to stop first")
```

Comment:
Can you also flip the order of these? I don't think the code even does this assertion because of the `Fatal`